packages/devtools_app/lib/src/ui/service_extension_widgets.dart

```diff
@@ -466,6 +461,90 @@ mixin _ServiceExtensionMixin<T extends _ServiceExtensionWidget> on State<T> {
   }
 }
 
+class BasicTooltip extends StatelessWidget {
+  const BasicTooltip({
+    Key key,
+    @required this.message,
+    @required this.child,
+  }) : super(key: key);
+
+  final String message;
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return Tooltip(
+      message: message,
+      waitDuration: tooltipWait,
+      preferBelow: false,
+      child: child,
+    );
+  }
+}
+
+/// Rich tooltip with a description and "more info" link
+class RichTooltip extends StatelessWidget {
+  const RichTooltip({
+    Key key,
+    @required this.message,
+    @required this.url,
+    @required this.child,
+  }) : super(key: key);
+
+  final String message;
+  final String url;
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return HoverCardTooltip(
+      enabled: () => true,
+      onHover: (_) => _buildCardData(context),
+      child: child,
+    );
+  }
+
+  Future<HoverCardData> _buildCardData(BuildContext context) {
+    return Future.value(
+      HoverCardData(
+        position: HoverCardPosition.element,
+        width: 300,
+        contents: Material(
+          child: Column(
+            mainAxisSize: MainAxisSize.min,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(message),
+              Align(
+                alignment: Alignment.bottomRight,
+                child: InkWell(
+                  onTap: () => launchUrl(url, context),
+                  borderRadius: BorderRadius.circular(defaultBorderRadius),
+                  child: Padding(
+                    padding: const EdgeInsets.all(8.0),
+                    child: Row(
+                      mainAxisSize: MainAxisSize.min,
+                      mainAxisAlignment: MainAxisAlignment.end,
+                      children: [
+                        Text(
+                          'More info',
+                          style: linkTextStyle(Theme.of(context).colorScheme),
+                        ),
+                        const SizedBox(width: 3),
+                        const Icon(Icons.launch, size: 12)
```

Comment:
try to use defaultIconSize from theme.dart. If that is too big, feel free to add another const in that file to represent the size here

Reply:
Done, added `tooltipIconSize`